server/app.py

```diff
@@ -216,6 +243,99 @@ async def submit_batch(batch: BatchData):
     return Response[BatchSubmitted](data=BatchSubmitted(job_id=batch.job_id))
 
 
+@app.post("/validate-connection", response_model=Response[ValidateConnectionResponse])
+async def validate_connection(request: ValidateConnectionRequest):
+    multi_model_provider_models = {
```

Comment:
could we name this something like `multi_model_provider_test_models`? Just so it's clear what they're being used for

Reply:
updated